start/cli/src/main/java/org/ethereum/beacon/command/RunDefaultSimulation.java

```diff
@@ -0,0 +1,40 @@
+package org.ethereum.beacon.command;
+
+import org.ethereum.beacon.SimulatorLauncher;
+import org.ethereum.beacon.emulator.config.ConfigBuilder;
+import org.ethereum.beacon.emulator.config.chainspec.Spec;
+import org.ethereum.beacon.emulator.config.simulator.SimulatorConfig;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "default",
+    description = "Runs simulation with default setup",
+    mixinStandardHelpOptions = true)
+public class RunDefaultSimulation implements Runnable {
+
+  @CommandLine.Option(
+      names = {"--loglevel"},
```

Comment:
What's the sense in `LogLevelAware` defined above this way?

Reply:
The intention was to share an option between two commands. But inheritance is not the best way of sharing a single option.

Reply:
Did you check [mixins](https://picocli.info/#_mixins) ?
If both inheritance and mixins doesn't look good for you, just remove `LogLevelAware`, let it go!

Reply:
ahaha